### PR TITLE
OO-715 New Styleguide

### PR DESCRIPTION
**Pull request by jiivonen**
Some refactoring still to do, but here are the main changes.

### CODE REVIEW

File: portfolio/src/app/directives/contactInformation/contactInformation.html [hunk 1]
```diff
@@ -58,10 +59,10 @@
     </div>
     <ul role="list">
       <li ng-repeat="someLink in contactInformation.someLinks | orderBy: 'type'"
-          role="listitem"
-          class="block link">
-        <a href="{{ someLink.url }}" tabindex="0" target="_blank">{{ 'contactInformation.' + someLink.type | translate }}
-          <i class="social-media-link-icon social-media-link-icon--{{ someLink.type | lowercase }}"></i>
+          role="listitem" class="">
```
Comment (greygriffin) [inline portfolio/src/app/directives/contactInformation/contactInformation.html]: Does this class="" serve some purpose, or can it be removed?

File: portfolio/src/app/directives/navigation/navigation.html [hunk 2]
```diff
@@ -17,14 +17,15 @@
...
   <div class="wrapper">
-    <nav>
-      <ul class="main-menu">
-        <li class="main-menu__home">
-          <a href="" ng-click="backToFrontpage()" aria-label="{{:: ('toolbar.home' | translate) }}"></a>
+    <nav class="main-menu">
+      <ul class="menu">
+        <li class="is-lvl1 main-menu__front">
+          <span class="main-menu__home"></span>
+          <a ng-click="backToFrontpage()" aria-label="{{:: ('toolbar.home' | translate) }}"></a>
```
Comment (greygriffin) [inline portfolio/src/app/directives/navigation/navigation.html]: Needs a tabindex="0"

File: portfolio/src/app/directives/preview/previewButton.html [hunk 3]
```diff
@@ -15,14 +15,16 @@
...
+          aria-labelledby="portfolio-preview-btn">
+    <span id="portfolio-preview-btn"
+          class="preview-btn--text"
+          translate="preview.enterPreview">
+    </span>
+    <span class="preview-btn--icon hy hy-preview"
+          aria-hidden="true">
+    </span>
+  </button>
+</div>
```
Comment (greygriffin) [inline portfolio/src/app/directives/preview/previewButton.html]: Missing final newline

File: portfolio/src/app/directives/navigation/navigation.html (hunk 2 above)
Comment (greygriffin) [inline portfolio/src/app/directives/navigation/navigation.html]: ...and was it an href=""?

File: portfolio/src/app/partials/_page_footer.html [hunk 4]
```diff
@@ -16,44 +16,62 @@
...
+           target="_blank" aria-label="Instagram"></a>
+      </div>
     </div>
 
-</div>
+  </div>
 
+  <div class="l-footer__copyright">
+    <p>
+      © {{::('general.helsinkiUniversity' | translate)}} {{currentYear}}
```
Comment (greygriffin) [inline portfolio/src/app/partials/_page_footer.html]: Fairly safe to have {{:: currentYear }}

File: portfolio/src/index.html [hunk 5]
```diff
@@ -33,7 +33,7 @@
     <title></title>
 
     <!-- build:css assets/styles/main.css -->
-    <link rel="stylesheet" href="assets/styles/styleguide/main.css">
+    <link rel="stylesheet" href="../bower_components/normalize.css/normalize.css"/>
```
Comment (greygriffin) [inline portfolio/src/index.html]: Should be "../../bower_components"

File: main/src/scss/main.scss [hunk 6]
```diff
@@ -21,7 +21,7 @@
 @import "../bower_components/singularity/stylesheets/singularitygs.scss";
 
 /* Styleguide elements required by the application */
-@import "styleguide/styleguide";
+@import "../common/src/scss/styleguide/styleguide";
```
Comment (greygriffin) [inline main/src/scss/main.scss]: Does this need an extra ../ ? From this file's location, common is ../../common . But perhaps the path is evaluated in some other context? (The corresponding file in portfolio has the same puzzle).

Comment (jiivonen) [inline main/src/scss/main.scss]: Afaik the grunt-sass runs it in the main. So both ../common (related to main) and ../../../common (related to the main.scss) work.

**Pull request merged**